go-controller/pkg/testing/exec.go

```diff
@@ -69,6 +71,10 @@ func (f *FakeExec) ErrorDesc() string {
 
 func (f *FakeExec) internalErrorDesc() string {
 	desc := "Executed commands do not match expected commands!\n"
+	if f.receivedUnexpected {
+		desc += fmt.Sprintf("Received unexpected command %s\n\n", f.unexpectedCommand)
```

Comment:
Nit: `s/Received/Executed/`, feel like it's a bit more explicit that the test executed something that had not been added to the test by the developer.